tools/wheels/cibw_before_build_macos.sh

```diff
@@ -21,17 +21,26 @@ if [[ $PLATFORM == "macosx-x86_64" ]]; then
   #GFORTRAN=$(type -p gfortran-9)
   #sudo ln -s $GFORTRAN /usr/local/bin/gfortran
   # same version of gfortran as the openblas-libs and scipy-wheel builds
-  curl -L https://github.com/MacPython/gfortran-install/raw/master/archives/gfortran-4.9.0-Mavericks.dmg -o gfortran.dmg
-  GFORTRAN_SHA256=$(shasum -a 256 gfortran.dmg)
-  KNOWN_SHA256="d2d5ca5ba8332d63bbe23a07201c4a0a5d7e09ee56f0298a96775f928c3c4b30  gfortran.dmg"
+  curl -L https://github.com/isuruf/gcc/releases/download/gcc-11.3.0-2/gfortran-darwin-x86_64-native.tar.gz -o gfortran.tar.gz
+
+  GFORTRAN_SHA256=$(shasum -a 256 gfortran.tar.gz)
+  KNOWN_SHA256="981367dd0ad4335613e91bbee453d60b6669f5d7e976d18c7bdb7f1966f26ae4  gfortran.tar.gz"
   if [ "$GFORTRAN_SHA256" != "$KNOWN_SHA256" ]; then
       echo sha256 mismatch
       exit 1
   fi
 
-  hdiutil attach -mountpoint /Volumes/gfortran gfortran.dmg
-  sudo installer -pkg /Volumes/gfortran/gfortran.pkg -target /
-  otool -L /usr/local/gfortran/lib/libgfortran.3.dylib
+  sudo mkdir -p /opt/
+  # places gfortran in /opt/gfortran-darwin-x86_64-native. There's then
+  # bin, lib, include, libexec underneath that.
+  sudo tar -xv -C /opt -f gfortran.tar.gz
+  sudo mkdir -p /usr/local/gfortran/lib
+  sudo cp -R /opt/gfortran-darwin-x86_64-native/lib/ /usr/local/gfortran/lib
+  ln -sf /opt/gfortran-darwin-x86_64-native/bin/gfortran /usr/local/bin/gfortran
```

Comment:
You probably want to do https://github.com/MacPython/gfortran-install/blob/47bb0df1e6f766503f81e4fd3d5d2acadd887393/gfortran_utils.sh#L125-L130 instead

Reply:
Can do. However, the overall effect seems quite similar? What are the advantages of the suggestion? (coming at this from a learning POV)

Reply:
`/usr/local/lib` is in the linker and the dynamic loader's search path by default. So you probably don't have to mess with `LIBRARY_PATH` and `DYLD_LIBRARY_PATH`.